docs/en/latest/profile.md

```diff
@@ -57,15 +57,27 @@ export APISIX_NODE_LISTEN=8132
 export DEPLOYMENT_ADMIN_ADMIN_LISTEN=9232
 ```
 
-If you don't set these environment variables, the configuration will use the default values provided.
+To use default values if no environment variables are set, you can add it to your configuration file as shown below:
+
+```yaml title="config.yaml"
+apisix:
+  node_listen:
+    - ${{APISIX_NODE_LISTEN:=9080}}                 
+deployment:
+  admin:
+    admin_listen:
+      port: ${{DEPLOYMENT_ADMIN_ADMIN_LISTEN:=9180}} 
+```
 
```

Comment:
Likewise, please sum up

Reply:
Hey @navendu-pottekkat, perhaps you did miss this request?

Reply:
@navendu-pottekkat please check this